gapic/src/main/com/google/gapid/models/Settings.java

```diff
@@ -164,12 +165,8 @@ public void addToRecent(String file) {
   }
 
   public String[] getRecent() {
-    return stream(recentFiles)
-        .map(file -> new File(file))
-        .filter(File::exists)
-        .filter(File::canRead)
-        .map(File::getAbsolutePath)
-        .toArray(l -> new String[l]);
+    return stream(recentFiles).map(file -> new File(file)).filter(File::exists)
```

Comment:
I think the previous version was more readable, can you please keep each pipeline stage on its own line?